models/public/quartznet/model.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import torch
+import yaml
```

Comment:
I think depending on PyYAML here is a bit fragile. It's not actually declared as a conversion dependency; the only reason it's available in the conversion environment is because it's a dependency of yacs, which is a dependency of another model (`regnetx-3.2gf`). So if we remove that model, we might end up removing PyYAML as well.

I would suggest to instead add the ruamel.yaml wheel to the downloaded files (it's pure Python) and use that here. It'll also let you drop a couple of regex hacks.

Reply:
Done